### PR TITLE
CSS `opacity` based on `prefers-reduced-transparency` media query

### PR DESCRIPTION
**Pull request by ramiy**
### Description

Add an example that demonstrates how to use the `prefers-reduced-transparency` media query to specify the desired `opacity` based on the user's preferences.

### CODE REVIEW

**Comment by ramiy** [comment]
@chrisdavidmills, this PR is similar to https://github.com/mdn/content/pull/30712.

**Review by estelle — COMMENTED** [review]
I suggested 1) edits to see also and 2) moving the mention from the description section to the a11y section.


I am not sure if you want to expand on this PR or if we should open an issue, but this page could use some work completely unrelated to the PRT issue. 

Stuff i would love to see  
* Even though the element is not visible, it is still part of the DOM. If it was in the tabindex order when visible it will still be there. We do mention pointer events. But we should tell people to make the content visible when focused.
* Opacity alone should not be used to provide information to screen readers. Use the `hidden` attribute or `visibility` or `display` style properties. It's best to avoid using `aria-hidden` attribute, but if hidden with opacity, hide it from screen readers too.
* a mention of setting opacity as a value for [@starting-style](https://developer.mozilla.org/en-US/docs/Web/CSS/@starting-style) when transitioning from display none to a visible value.
* a comment in the stacking context paragraph about how adding opacity can improve perf, with a link to where that's described.
* maybe a link to a drag/drop example as opacity is commonly used on items being dragged.

File: files/en-us/web/css/opacity/index.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ To change the opacity of a background only, use the {{cssxref("background")}} pr
 background: rgba(0, 0, 0, 0.4);
 ```
 
+Various operating systems provide a preference for reducing transparency. To set the `opacity` based on the user's operating systems transparency preferences, use the [`prefers-reduced-transparency`](/en-US/docs/Web/CSS/@media/prefers-reduced-transparency) media query.
```
[the remark this answers is not in the document]
Comment (ramiy) [reply files/en-us/web/css/opacity/index.md]: done

**Comment by ramiy** [comment]
@estelle I prefer no to expand on this PR.

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**